modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaConfigurationModelParser.java

```diff
@@ -69,6 +71,7 @@ public JavaConfigurationModelParser(JavaExtensionModelParser extensionModelParse
     this.configElement = configElement;
 
     parseStructure();
+    this.minMuleVersion = getMinMuleVersion().orElse(FIRST_MULE_VERSION);
```

Comment:
the agreement on the call was that all instances of this would have a TODO pointing to the issue that tracks using the MMV as a baseline

Reply:
adding it